backoffice/views/templates/nsButtons.html

```diff
@@ -6,9 +6,15 @@
         <div class="col-sm-7 col-sm-offset-3 text-right" style="margin-bottom: 1%; display: inline;">
             <button style="pointer-events:auto" type="submit" class="btn btn-success" ng-disabled="disableSave"><span translate>ns.save</span><i class="fa fa-floppy-o" aria-hidden="true"></i></button>
             <button style="pointer-events:auto" type="button" class="btn btn-default btn-s&q" ng-click="saveAndQuit()" ng-disabled="disableSave" ng-hide="hideSaveAndQuit" translate>ns.saveQuit</button>
-            <button ng-repeat="btn in additionnalButtons" style="pointer-events:auto" type="button" class="btn btn-default" ng-class="btn.class" ng-click="btn.onClick()" ng-style="btn.style"><span translate="{{btn.text}}"></span><i class="fa fa-eye" aria-hidden="true"></i></button>
+            <button ng-repeat="btn in additionnalButtons" style="pointer-events:auto" type="button" class="btn btn-default" ng-class="btn.class" ng-click="btn.onClick()" ng-style="btn.style"><span translate="{{btn.text}}"></span><span bind-html-compile="btn.moreText"></span></button>
             <button style="pointer-events:auto" type="button" class="btn btn-default" ng-click="duplicateProduct()" ng-if="isEditMode && duplicateProduct"><span translate>ns.dup</span><i class="fa fa-clone" aria-hidden="true"></i></button>
             <button style="pointer-events:auto" type="button" class="btn btn-danger" ng-click="remove()" ng-if="isEditMode" ng-hide="hideRemove" ><span translate>ns.delete</span><i class="fa fa-trash" aria-hidden="true"></i></button>
+            <span ng-if="moreButtons">
+                <button style="pointer-events:auto;padding-top:3px;min-width:0px;" type="button" class="btn btn-default" ng-click="(toto.display == 'block' ? toto.display = 'none': toto.display = 'block' );( toto.display == 'block' ? moreButton.style = 'fa-minus' : moreButton.style = 'fa-plus')" ><i class="fa {{moreButton.style || 'fa-plus'}} fa-2x" style="visibility:visible!important;display:inline-block!important" ng-style="btn.dispStyle" aria-hidden="true"></i></button>
```

Comment:
I don't see where toto is used but instead of doing two ternary operations, you should use a ng-show or a ng-class with the two styles